packages/paste-core/components/card/__test__/card.test.tsx

```diff
@@ -65,3 +73,30 @@ describe('Card', () => {
     expect(results).toHaveNoViolations();
   });
 });
+
+describe('HTML attributes', () => {
+  it('should set a element data attribute for Card', () => {
+    testRender(<Card>card-content</Card>);
+    expect(screen.getByText('card-content').getAttribute('data-paste-element')).toEqual('CARD');
+  });
+  it('should set a custom element data attribute for Card', () => {
+    testRender(<Card element="foo">card-content</Card>);
+    expect(screen.getByText('card-content').getAttribute('data-paste-element')).toEqual('foo');
+  });
+});
+
+describe('Customization', () => {
+  it('should add custom styles to Card', (): void => {
+    testRender(
+      <CustomizationProvider
+        baseTheme="default"
+        elements={{CARD: {backgroundColor: 'colorBackground', borderColor: 'colorBorderDestructive'}}}
+      >
+        <Card data-testid="customizable-card">Custom card</Card>
+      </CustomizationProvider>
+    );
+    const renderedCard = screen.getByTestId('customizable-card');
+    expect(renderedCard).toHaveStyleRule('background-color', 'rgb(244,244,246)');
+    expect(renderedCard).toHaveStyleRule('border-color', 'rgb(214,31,31)');
+  });
+});
```

Comment:
Should we also test that customization should add custom styles to Card when a custom `element` attribute value is being used? Or is that redundant?

Reply:
Yeah. I can add a test for that also.